attest/net/src/ias.rs

```diff
@@ -13,8 +13,8 @@ use mc_util_encodings::{FromBase64, FromHex, ToBase64};
 use pem::parse_many;
 use percent_encoding::percent_decode;
 use reqwest::{
+    blocking::Client,
```

Comment:
This stuff was breaking during `make dist` - I have a ticket to see why this isn't exercised as part of CI.